launch.go

```diff
@@ -176,7 +171,7 @@ func launch(api screwdriver.API, buildID, rootDir, emitterPath string) error {
 		return fmt.Errorf("fetching Pipeline ID %q: %v", j.PipelineID, err)
 	}
 
-	scm, err := parseScmUri(p.ScmUri, p.ScmRepo.Name)
+	scm, err := parseScmURI(p.ScmUri, p.ScmRepo.Name)
```

Comment:
should we change p.ScmUri to p.ScmURI to be consistent?
https://github.com/screwdriver-cd/launcher/blob/master/screwdriver/screwdriver.go#L94